backend/lib/models/Comment.js

```diff
@@ -65,7 +65,7 @@ function updateAuthorType(authorID, newAuthorType) {
 
 module.exports = {
   model: Comment,
-  schema: CommentSchema,
+  schema: commentSchema,
```

Comment:
Note that this commit is actually part of this [PR](https://github.com/FightPandemics/FightPandemics/pull/586). I need to include this commit so that the backend service is able to run.